### ISSUE TITLE
JSON error when editing event group information

### ISSUE THREAD

**Issue by JChipault** [USGS-WiM/whispersservices_django#379]
I'm hitting some JSON errors on the test site in both firefox and chrome when I try to edit event groups in my dashboard (change type, remove an event, or edit event group comment). I checked and I'm not logged into the API.

Here's firefox:
![image](https://user-images.githubusercontent.com/49287076/81718769-c52bf980-9441-11ea-94c7-0c5c94e62c92.png)

and a slightly different message if I try in chrome:
![image](https://user-images.githubusercontent.com/49287076/81718827-d7a63300-9441-11ea-9892-ef3181da393d.png)

**Comment by BlakeDraper** [comment]
@aaronstephenson any ideas on this one? never seen this message before. 

>MultipleObjectsReturned at /eventgroups/4/
get() returned more than one EventEventGroup -- it returned 2!

![image](https://user-images.githubusercontent.com/6978380/81987750-e5091c00-95ff-11ea-93af-c3609f3e4ec6.png)

```python
Traceback:

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/django/core/handlers/exception.py" in inner
  34.             response = get_response(request)

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/django/core/handlers/base.py" in _get_response
  115.                 response = self.process_exception_by_middleware(e, request)

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/django/core/handlers/base.py" in _get_response
  113.                 response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/django/views/decorators/csrf.py" in wrapped_view
  54.         return view_func(*args, **kwargs)

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/rest_framework/viewsets.py" in view
  116.             return self.dispatch(request, *args, **kwargs)

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/rest_framework/views.py" in dispatch
  509.             response = self.handle_exception(exc)

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/rest_framework/views.py" in handle_exception
  469.             self.raise_uncaught_exception(exc)

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/rest_framework/views.py" in raise_uncaught_exception
  480.         raise exc

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/rest_framework/views.py" in dispatch
  506.             response = handler(request, *args, **kwargs)

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/rest_framework/mixins.py" in update
  68.         self.perform_update(serializer)

File "/var/www/whispersservices_django/whispersservices/views.py" in perform_update
  148.             serializer.save(modified_by=self.request.user)

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/rest_framework/serializers.py" in save
  199.             self.instance = self.update(self.instance, validated_data)

File "/var/www/whispersservices_django/whispersservices/serializers.py" in update
  2207.             comment.save()

File "/var/www/whispersservices_django/whispersservices/models.py" in save
  1938.             event = EventEventGroup.objects.get(eventgroup=self.object_id).event

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/django/db/models/manager.py" in manager_method
  82.                 return getattr(self.get_queryset(), name)(*args, **kwargs)

File "/var/www/whispersservices_django/env/lib/python3.5/site-packages/django/db/models/query.py" in get
  412.             (self.model._meta.object_name, num)
```

```json
{
  "id": 4,
  "category": 1,
  "new_comment": "2nd test event group. no real significance. comment edited once. comment edited again.",
  "new_events": [
    170217,
    170216
  ]
}
```

also odd that if I go to that eventgroup records, everything seems fine:
![image](https://user-images.githubusercontent.com/6978380/81988063-74aeca80-9600-11ea-8223-254b7c6a8f41.png)
any my comment edit did stick, tho the server errored as if it had not

**Comment by aaronstephenson** [comment]
The error is occurring in a code block that updates the Event modified_date and modified_by fields, after all the saves have completed, which is why all the objects exist when you query them.

Fixed in 4921fd35

**Comment by JChipault** [comment]
working now for me. Thanks!
